packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/protocol-properties.md

```diff
@@ -0,0 +1,14 @@
+| <div style="width:150px">Property</div> | &nbsp; | Description  |
+| ----------------- | --: | ------------ |
+| **Provide your default ACS URL** `*` | | Specify the default Assertion Consumer Service (ACS) URL. If you're using a per tenant design, include the variable names that you created in your URL. For example: `https://${org.subdomain}.example.org/strawberry/login`<br>The maximum field length is 1024 characters. |
+| **Provide any additional ACS URLs** | | Provide an indexed ACS URL if your integration supports multiple ACS URLs where the SAML response can be sent.<br>Adding an ACS URL is optional, but if you add one, both `ACS URL` and `Index` properties are required.<br>Click **+ Add another** to add another URL.|
```

Comment:
Who is sending the SAML response? You could avoid the "be" verb if you reword to: Provide an indexed ACS URL if your integration supports multiple ACS URLs where ?? sends the SAML response.

```suggestion
| **Provide any additional ACS URLs** | | Provide an indexed ACS URL if your integration supports multiple ACS URLs where the SAML response can be sent.<br>Adding an ACS URL is optional, but if you add one, both `ACS URL` and `Index` properties are required.<br>Click **+ Add another** to add another URL.|
```

Reply:
Thanks! I confirmed with Jack that I can use "Okta" as the sending entity. I'll update this in another commit.